src/lsp/cobol_config/cobol_config.ml

```diff
@@ -57,7 +57,8 @@ let default_search_path =
       if Sys.(win32 || cygwin)
       then append ~sub:"SuperBOL" (Sys.getenv_opt "APPDATA") @
            append ~sub:"SuperBOL" (Sys.getenv_opt "LOCALAPPDATA")
-      else []
+      else ["/usr/local/share/gnucobol/config";
+            "/usr/share/gnucobol/config"]
     in
```

Comment:
Actually on Cygwin you *might* want to have both `*APPDATA` and `*/share/gnucobol/config` (you don't know if the user installed a "Cygwin" GnuCOBOL or a native GnuCobol).

Reply:
It also depends what kind of superbol is installed though. A Windows-native superbol has no idea about Unix paths, while a Cygwin superbol can handle both Unix and Windows paths...

Reply:
Not even mentionning MSYS2, which builds native binaries (can't be distinguished with `Sys.*`) that can use both Windows-style and Unix-style paths.

Reply:
Hum… do you think it is worth hacking with a check using `EzFile.FileOS.dir_separator = "/"`?

Reply:
I have no idea how `EzFile.FileOS.dir_separator` behaves when run from a Cygwin or MSYS2 shell.

I ran a few tests to clarify the behavior of Cygwin and MSYS2 with respect to paths. Turns out I was partly wrong.
Here's what I learnt :
- native Windows binaries built by Cygwin or MSYS2 have no idea about Unix paths
- "Cygwin" binaries built by Cygwin understand both Windows and Unix paths
- MSYS2 can also produce "Cygwin" binaries, though people don't do that (the whole point of MSYS2 is to build native binaries)

So basically, when you have `Sys.cygwin = true`, then you can add both `*APPDATA` and `*/share/gnucobol/config` to your search path (even if it seems unlikely people will ever run a Cygwin superbol server).

Reply:
Ok I've amended to take cygwin into considerations.

Reply:
Perfect !